README.md

```diff
@@ -54,6 +53,34 @@ You can view example usage in [REFERENCE](REFERENCE.md).
 
 See [REFERENCE](REFERENCE.md).
 
+## Limitations
+
+If running CentOS/RHEL, ensure the `epel` repo, or another repo containing a
+suitable Erlang version, is present. On Debian systems, puppetlabs/apt
+(>=2.0.0 < 5.0.0) is a soft dependency.
+
+To have a suitable erlang version installed on RedHat and Debian systems,
+you have to install another puppet module from http://forge.puppetlabs.com/garethr/erlang with:
+
+    puppet module install garethr-erlang
+
+This module handles the packages for erlang.
+To use the module, add the following snippet to your site.pp or an appropriate profile class:
+
+For RedHat systems:
+
+    include 'erlang'
+    class { 'erlang': epel_enable => true}
```

Comment:
You can get syntax highlighting
```suggestion
```puppet
include 'erlang'
class { 'erlang': epel_enable => true}
```

(Note Github doesn't allow me to suggest the last closing backticks)

The same goes below for the Debian suggestion.